Modules/posixmodule.c

```diff
@@ -3252,36 +3300,8 @@ posix_getcwd(int use_bytes)
     PyObject *obj;
 
 #ifdef MS_WINDOWS
-    if (!use_bytes) {
-        wchar_t wbuf[MAXPATHLEN];
-        wchar_t *wbuf2 = wbuf;
-        PyObject *resobj;
-        DWORD len;
-        Py_BEGIN_ALLOW_THREADS
-        len = GetCurrentDirectoryW(Py_ARRAY_LENGTH(wbuf), wbuf);
-        /* If the buffer is large enough, len does not include the
-           terminating \0. If the buffer is too small, len includes
-           the space needed for the terminator. */
-        if (len >= Py_ARRAY_LENGTH(wbuf)) {
-            wbuf2 = PyMem_RawMalloc(len * sizeof(wchar_t));
-            if (wbuf2)
-                len = GetCurrentDirectoryW(len, wbuf2);
-        }
-        Py_END_ALLOW_THREADS
-        if (!wbuf2) {
-            PyErr_NoMemory();
-            return NULL;
-        }
-        if (!len) {
-            if (wbuf2 != wbuf)
-                PyMem_RawFree(wbuf2);
-            return PyErr_SetFromWindowsErr(0);
-        }
-        resobj = PyUnicode_FromWideChar(wbuf2, len);
-        if (wbuf2 != wbuf)
-            PyMem_RawFree(wbuf2);
-        return resobj;
-    }
+    if (!use_bytes)
```

Comment:
`os.getcwdb` is also incorrect. It shouldn't call the CRT's ANSI function. It needs to call `win32_wgetcwd` and convert the result to the file-system encoding, which defaults to UTF-8. This was apparently overlooked when implementing PEP 529.

Reply:
Indeed. According to PEP 529, `os.getcwdb` should have also been undeprecated. I'll file an issue for that in a while.